fun/Wheel of Fortune.go.tmpl

```diff
@@ -0,0 +1,54 @@
+{{/*
+	Name: Wheel of Fortune
+	Author: Dr. Hyakkimaru#6020
+	
+	Recommended:
+	- Trigger Type: Regex
+	- Trigger: `\A\-(?i)wheel(offortune)?`
+
+	Copyright: 2021 Hyakki999
+	License: MIT
+	Repository: https://github.com/Hyakki999/YAGPDB-cc
```

Comment:
We have a different copyright notice, so the first part isn't really accurate. I'd also prefer that the repository link points to here (but you may also add something like "also available here on the author's own repo [link]").

As such, I would prefer something like this:
```suggestion
	Author: Hyakki999 <https://github.com/Hyakki999>
	License: MIT
```

Reply:
Fixed the first part.